src/pybind/mgr/volumes/module.py

```diff
@@ -310,6 +310,12 @@ class Module(orchestrator.OrchestratorClientMixin, MgrModule):
             'desc': "List NFS Clusters",
             'perm': 'r'
         },
+        {
+            'cmd': 'nfs cluster info '
+                   'name=clusterid,type=CephString ',
```

Comment:
maybe make the `clusterid` param optional and show info for all NFS daemons ?

Reply:
Done